docs/topics/proc_deploying-the-circuit-breaker-booster-using-the-oc-cli-client.adoc

```diff
@@ -5,11 +5,12 @@
 // Rationale: This procedure is identical in all deployments.
 
 [id='deploying-the-circuit-breaker-booster-using-the-oc-cli-client_{context}']
-= Deploying the {name-mission-circuit-breaker} booster using the `oc` CLI client
+[id='deploying-the-circuit-breaker-example-using-the-oc-cli-client_{context}']
+= Deploying the {name-mission-circuit-breaker} example using the `oc` CLI client
```

Comment:
Deploying the {name-mission-circuit-breaker} example using --> Deploying the {name-mission-circuit-breaker} example application using

Reply:
Yes!